.pre-commit-config.yaml

```diff
@@ -1,3 +1,5 @@
+default_language_version:
```

Comment:
We are not including this part in this (or other libraries) for now as I believe it is a bit setup specific. Can you remove it from the PR and have this change only locally?

Reply:
Done